README.md

```diff
@@ -1,70 +1,73 @@
+web. We aim to enable individual state and library-based newspaper collections
+to more easily provide these core capabilities, as well as develop new common
+features and improvements.
+
+Read ["About Open ONI" in our
+wiki](https://github.com/open-oni/open-oni/wiki/About-Open-ONI) for more
+information about the project's origins.
+
+## Open ONI-powered Sites
+Visit sites currently powered by Open ONI:
+
+- [Historic Oregon Newspapers](https://oregonnews.uoregon.edu/)
+- [Pennsylvania Newspaper Archive](http://panewsarchive.psu.edu/)
+- [Nebraska Newspapers](https://nebnewspapers.unl.edu/)
+
+View ["Sites using Open ONI" in our
+wiki](https://github.com/open-oni/open-oni/wiki/Sites-Using-Open-ONI) for links
+to the sites' source code.
+
+## Install
+To evaluate the software, we recommend installing from the [latest Open ONI
+release](https://github.com/open-oni/open-oni/releases/latest), rather than
+checking out the latest changes on the `master` branch.
+
+If you're interested in more recent changes, check out the `dev` branch but be
+warned that we don't expect it will always be in a working state at this point.
+
+## Documentation
+Documentation is available in the [Open ONI
+wiki](https://github.com/open-oni/open-oni/wiki).
+
+[Production install
+instructions](https://github.com/open-oni/open-oni/tree/dev/docs) are in
+progress on the `dev` branch.
+
+## Support
+Open ONI is currently not ready for general production use outside the
+institutions who have been able to provide time for developer participation and
+have deployed it individually. Documentation and changes to facilitate more
+standardized deployment are under active development, but maintainers can only
+provide a minimal amount of assistance while working on Open ONI part-time.
+Developers cannot be expected to provide extensive one-on-one support.
+
+We are glad to provide support specifically related to Open ONI, but anyone
+wanting to work with Open ONI should begin with some foundational knowledge of
+Linux, Docker, Apache, Python, Django, Solr, MariaDB/MySQL, and IIIF. There are
+other resources and communities better suited to aid in getting started and
+familiar with those technologies, so general assistance with them falls outside
+the scope of what to expect Open ONI developers to provide.
+
+We ask that bugs and feature requests be submitted to [Open ONI on
+GitHub](https://github.com/open-oni/open-oni/issues). Please confine other
+questions and conversation about Open ONI development and use to [Open ONI's
+Slack](http://bit.ly/openoni-slack-signup). These guidelines are intended both
+to keep conversation in the open where it may benefit all of the Open ONI
+community and to protect developers' already limited time.
+
+## Contact Us
+We prefer to communicate on the [Open ONI
+Slack](http://bit.ly/openoni-slack-signup), but if you have trouble or prefer
+not to use Slack, email Karin Dalziel (kdalziel2 [at] unl.edu).
```

Comment:
@techgique should we change this to you? (I don't mind being the contact person still if you don't want to)

Reply:
We'll revisit this soon after some more community organization planning decisions are made :)